src/Nethermind/Nethermind.Synchronization/ParallelSync/SyncMode.cs

```diff
@@ -21,7 +21,10 @@ namespace Nethermind.Synchronization.ParallelSync
     [Flags]
     public enum SyncMode
     {
-        None = 0,
+        /// <summary>
+        /// We are connected to nodes and processing based on discovery
+        /// </summary>
+        Idle = 0,
```

Comment:
this is not following the rules on Flags where you are supposed to have None and All (this is used often in auto code completion - also it may cause trouble when you do checks like:
(Idle & modes) == Idle